packages/massa-proto-cli/src/index.ts

```diff
@@ -4,7 +4,10 @@ import { generateAsCallers } from './as-gen';
 import { generateTsCallers } from './ts-gen';
 import { ProtoFile, getProtoFiles, getProtoFunction } from './protobuf';
 import { Command } from 'commander';
-
+import {
+  fetchCustomTypes,
+  MassaCustomType,
+} from '@massalabs/as-transformer/dist';
```

Comment:
out of curiosity, why `dist` ? it's not the case for other packages.

Reply:
Because this package only exports his built version, its not meant to be used as a library since its a cli, however I added some exported built types to it to be used here.